### DISCUSSION TITLE
Docker single image

### DISCUSSION THREAD

**Discussion by aderici** [Budibase/budibase#5456] [Help] [answered]
has anyone successfully published a single docker image version of budibase? I see there has been some work recently but couldnt find a published image on docker hub budibase account

**Comment by melohagan** [comment, marked as answer]
Hi @aderici 

This will be worked on in the future: https://github.com/Budibase/budibase/issues/5202

**Reply by aderici** [reply]
Thank you for the information, will be looking forward to it,
